src/components/molecules/sort-picker/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Picker, StyleSheet } from 'react-native';
+
+export const SortPickerComponent = ({ onSortChange: handleSortChange, currentSort, items = [] }) => {
```

Comment:
If `items` has a default prop of `[]`, can it be in `defaultProps` or can there be a consistent pattern for defining the `defaultProps`?

Reply:
There are quite a few style guide things like this which we should define. Another is when to use prop types - I've been adding them, but most components don't have them and it's not required by lint, so it's inconsistent at the moment. 

Make an issue for this and pin it down there?

Reply:
Sorry, this file was removed !